server/src/main/scala/org/ensime/server/Server.scala

```diff
@@ -115,13 +117,23 @@ object Server extends AkkaBackCompat {
   Slf4jSetup.init()
 
   val log = LoggerFactory.getLogger("Server")
+  val XDG_CONFIG_HOME = " ~/.config/ensime-server"
```

Comment:
I'm ok with the xdg home, then we don't need 3.

This does mean the project root becomes relevant again, so you can remove the deprecation warning / comment that I added in `config.scala`